Render Jobs Page development.

ZaykaNya opened the pull request:
# Pull Request ☁️
## Overview ✨
<!-- Please give a short summary about pull request changes. -->
Created interface and entity for render jobs. Connected Render Jobs Page with core.

## Changes 🔧
<!-- Please describe what did you do. Use closing keyword to close issues. Write module in brackets [] -->
- __[Jobs]__ _Created interface for render jobs._
- __[ShortJobs]__ _Created entity for render jobs._
- __[RenderJobs]__ _Connected with core._

## Additional context 🌟
<!-- Any additional context -->

ZaykaNya committed:
[Jobs] Created interface for render jobs.
[ShortJobs] Created entity for render jobs.
[RenderJobs] Connected with core.

DanilAndreev commented on src/core/validators/DateValidator.ts:
```diff
@@ -25,6 +25,7 @@ export default function DateValidator(value: any): ValidatorResult<Date> {
     }
     let valueCopy = "" + value;
     if (isNaN(Date.parse(valueCopy))) {
+        console.log(Date.parse("hi" + valueCopy));
```
Remove unused console logs.

ZaykaNya committed:
[DateValidator] (PR #131) Deleted console.log().
[RenderJob] Changed entity.
[RenderJobsDetails] Created functions to get data from server.
